book/applicaster-video-platform/js2native/readme.md

```diff
@@ -143,69 +143,54 @@ TVShowID: "1",
 ```
  #### Morpheus.updateUserProfile()
  Update user profile information. The old properties will be overwritten by the new ones.
+ 
  **userProperties** - The user properties information. The keys of the
-JSON object should be formatted as pascal case (*PascalCase*). Morevoer,
-properties which are not *custom properties* are called *special properties*.
-They are not required, but if delivered, should be written as follows:
- * Name
+JSON object should be formatted as pascal case (*PascalCase*). 
+Moreover, properties which are not *custom properties* are called *special properties*. They are not required, but if delivered, should be written in the following format: "CustomFieldName".
```

Comment:
I'd remove this line. No longer really relevant. This was from back in the days prior to plugins when FB sent us "special properties"